data/beaconrestapi/src/main/java/tech/pegasys/teku/beaconrestapi/JsonTypeDefinitionBeaconRestApi.java

```diff
@@ -242,6 +243,7 @@ private static RestApi create(
             .endpoint(new GetAttesterSlashings(dataProvider, spec))
             .endpoint(new GetAttesterSlashingsV2(dataProvider, schemaCache))
             .endpoint(new PostAttesterSlashing(dataProvider, spec))
+            .endpoint(new PostAttesterSlashingV2(dataProvider, schemaCache))
```

Comment:
we could even try deprecating here, i think potentially if hte new one is first, it should come up before the one we're killing off..

Reply:
deprecated the V1. Changing the order here didn't make any difference on swagger at least